pymc4/gp/_kernel.py

```diff
@@ -0,0 +1,125 @@
+"""Constant Kernel and WhiteNoise Kernel."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow.compat.v2 as tf  # pylint: disable=import-error
```

Comment:
Do we still need to use `compat.v2`?

Reply:
Oh sorry! Was using that for consistency with tfp's convention. I should be changed now.